decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
Ok, so I should include the previous diff in the PR, right? Note that this means we'll have to edit this controller everytime we add something to the main layout at the controller level. Really undecided between that and `< ApplicationController` + `skip_authorize_resource`...